subsys/usb/usb_c/usbc_pe_common.c

```diff
@@ -263,17 +296,15 @@ void pe_report_error(const struct device *dev, const enum pe_error e,
 	 *     Message is received instead of the expected GoodCRC Message
 	 *     response.
```

Comment:
Move this comment block to the helper function.

Reply:
Done

Reply:
Note this is done with https://github.com/zephyrproject-rtos/zephyr/pull/52993/commits/cdc2e97f749fc203f231bc7db548a5aa0062ba52